qa/suites/rados/basic/tasks/rgw_snaps.yaml

```diff
@@ -8,8 +8,8 @@ overrides:
         osd_max_omap_entries_per_request: 10
 tasks:
 - rgw:
-    default_idle_timeout: 3600
-    client.0: null
+    client.0:
+      default_idle_timeout: 3600
```

Comment:
the `default_idle_timeout` field was global to the rgw task (see https://github.com/ceph/ceph/pull/15184/files#diff-995b04809fcabacc3e3ecfaea903a41aL1128). there was also an `idle_timeout` that could be used per-client

those global fields are removed from `config` as they're parsed, resulting in a `config` whose only keys are clients. that's why it's now being mistakenly interpreted as a remote in this test failure. i'd expect the same thing to happen in the upgrade tests

moving the line under client.0 does work to pass the test - it just won't be used anywhere